Sources/Utils.swift

```diff
@@ -143,4 +143,5 @@ extension Collection where Index: Comparable, Self.Indices.Index == Index {
   
 #else
   import Foundation
+  typealias RecursiveLock = NSRecursiveLock
```

Comment:
Yes I'm afraid so 😞. Unless I'm missing something?

This is using Swift 3 on Xcode Beta 4.

Sent from my iPhone

On Aug 3, 2016, at 5:03 PM, Ilya Puchka <notifications@github.com<mailto:notifications@github.com>> wrote:


In Sources/Utils.swift<https://github.com/AliSoftware/Dip/pull/106#discussion_r73420359>:

> @@ -143,4 +143,5 @@ extension Collection where Index: Comparable, Self.Indices.Index == Index {
>
>  #else
>    import Foundation
> +  typealias RecursiveLock = NSRecursiveLock


so "NS" is back again? 😫

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub<https://github.com/AliSoftware/Dip/pull/106/files/78c628c6fa26155b111696e992586e28bacf5cdd#r73420359>, or mute the thread<https://github.com/notifications/unsubscribe-auth/ARAjCVsqLkY4BKL1WTOaqtL1di4-Jcptks5qcQIDgaJpZM4Jbudw>.